src/parser.js

```diff
@@ -462,6 +463,14 @@ export default class Parser extends Stream {
 
```

Comment:
Since we will create a new major version, I would assume it makes sense to remove this code:

```js
if (typeof this.manifest.dateTimeString === 'undefined') {
                // PROGRAM-DATE-TIME is a media-segment tag, but for backwards
                // compatibility, we add the first occurence of the PROGRAM-DATE-TIME tag
                // to the manifest object
                // TODO: Consider removing this in future major version
                this.manifest.dateTimeString = entry.dateTimeString;
                this.manifest.dateTimeObject = entry.dateTimeObject;
              }
```

Reply:
Removed in the latest commit and fixed tests.